src/uu/hashsum/src/hashsum.rs

```diff
@@ -297,7 +297,7 @@ pub fn uumain(mut args: impl uucore::Args) -> UResult<()> {
     };
     let check = matches.contains_id("check");
     let tag = matches.contains_id("tag");
-    let nonames = if binary_name == "b3sum" || matches.try_contains_id("b3sum").is_ok() {
+    let nonames = if binary_name == "b3sum" || matches.try_contains_id("b3sum").unwrap_or(false) {
```

Comment:
I used unwrap_or(false) as the return value of try_contains_id is Result<bool, MatchesError>